app/assets/javascripts/discourse/app/initializers/keyboard-shortcuts.js

```diff
@@ -1,10 +1,14 @@
 import KeyboardShortcuts from "discourse/lib/keyboard-shortcuts";
 import Mousetrap from "mousetrap";
+import bindGlobal from "mousetrap-global-bind";
 
 export default {
   name: "keyboard-shortcuts",
 
   initialize(container) {
+    // Ensure mousetrap-global-bind is executed
+    bindGlobal;
```

Comment:
Do we care about lgtm-com bot warning on this line?

Reply:
yes, I'm pushing a fix.